godtools/ViewControllers/Platform/LanguagesTableViewController.swift

```diff
@@ -17,6 +17,13 @@ class LanguagesTableViewController: BaseViewController {
     
     let languagesManager = LanguagesManager.shared
     
+    var screenTitleAux: String?
+    override var screenTitle: String {
+        get {
+            return screenTitleAux!.localized
```

Comment:
is `screenTitleAux` guaranteed to be present? if not, this could crash un wrapping an optional that is nil.